src/analyitcs/fathom/fathom.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Loads a Fathom instance into the page
+ * @param siteId The site id to use
+ * @returns
+ */
+export function loadFathom(siteId: string) {
+  return new Promise<void>(resolve => {
+    const script = document.createElement('script')
+    script.setAttribute('data-site', siteId)
+    script.setAttribute('data-auto', 'true')
+    script.src = 'https://cdn.usefathom.com/script.js'
+    script.defer = true
+    script.onload = () => resolve
+    document.body.appendChild(script)
+  })
+}
```

Comment:
We can put the `siteId` in environment variable and inject the data-site from webpack. 

This may fail after my CSP pull request. Because we are injecting script without nonce property.

@adamazad Is that ok?

Reply:
Makes sense, we can do that.

Reply:
I have a made an attempt to add it. Send you a git patch. 
One thing I noticed is brave blocks the tracking. 😞